tools/clang/tools/dxclib/dxc.cpp

```diff
@@ -822,33 +822,38 @@ int DxcContext::MaybeRunExternalValidatorAndPrintValidationOutput(
                               !opts.OptDump && !opts.DumpDependencies &&
                               !opts.VerifyDiagnostics;
   bool needsValidation = produceFullContainer && !opts.DisableValidation;
```

Comment:
Note, I've been ignoring this for now but as we get closer to landing this we'll need to do a pass through and make sure we're following LLVM coding standards.

```suggestion
  bool NeedsValidation = ProduceFullContainer && !Opts.DisableValidation;
```